st2reactor/st2reactor/rules/filter.py

```diff
@@ -28,7 +28,7 @@
 
 
 class RuleFilter(object):
-    def __init__(self, trigger_instance, trigger, rule):
+    def __init__(self, trigger_instance, trigger, rule, extra_info=False):
```

Comment:
Would be good to add a docstring for `extra_info` argument and maybe also call it `include_extra_info` or smth to make it more clear what is going on :)